src/cca.jl

```diff
@@ -336,3 +340,113 @@ function fit(::Type{CCA}, X::AbstractMatrix{T}, Y::AbstractMatrix{T};
 
     return M::CCA
 end
+
+abstract type MultivariateTest <: HypothesisTest end
+
+struct WilksLambda <: MultivariateTest
```

Comment:
The convention in HypothesisTests is for `HypothesisTest` subtypes to have names ending in `Test`. Perhaps these should be `WilksLambdaTest`, etc. for consistency?